pkg/querier/parquet_queryable.go

```diff
@@ -181,9 +191,24 @@ type parquetQuerierWithFallback struct {
 
 	// metrics
 	metrics *parquetQueryableFallbackMetrics
+
+	limits *validation.Overrides
+	logger log.Logger
 }
 
 func (q *parquetQuerierWithFallback) LabelValues(ctx context.Context, name string, hints *storage.LabelHints, matchers ...*labels.Matcher) ([]string, annotations.Annotations, error) {
+	userID, err := tenant.TenantID(ctx)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if q.limits.QueryVerticalShardSize(userID) > 1 {
+		uLogger := util_log.WithUserID(userID, q.logger)
+		level.Warn(uLogger).Log("msg", "parquet queryable enabled but vertival sharding > 0. Falling back to the block storage")
```

Comment:
This should be > 1 not > 0.

Does vertical sharding also impact label values? I think it is only for metrics queries?

Reply:
Its not > 1 already?

Reply:
ALso removing from the other places.. and leaving only on the select.